includes/base/element-base.php

```diff
@@ -430,6 +444,11 @@ public function add_link_attributes( $element, array $url_control, $overwrite =
 	public function print_element() {
 		$element_type = $this->get_type();
 
+		if ( $this->should_render_shortcode() ) {
+			echo '[elementor-element data="' . esc_attr( base64_encode( wp_json_encode( $this->get_raw_data() ) ) ) . '"]';
```

Comment:
Do we need `base64_encode()`?

Reply:
@KingYes 
`echo do_shortcode('...');`?

Reply:
Yes! If not, we can get a problem with the parsing args in the shortcode mechanism.